packages/i18n/src/utils/lang.tsx

```diff
@@ -0,0 +1,33 @@
+import { IntlShape } from "react-intl";
+import { Fragment, ReactNode } from "react";
+
+import { commonMessages } from "../messages";
+import { Locales } from "../types";
+
+const Gray = ({ children }: { children: ReactNode }) => (
```

Comment:
Is it possible to reuse the `gray` formatter from packages/i18n/src/components/richTextElements.tsx instead of making a new one?  I see that it's structured a little differently so maybe not...  Could you add a comment that it should stay in sync, at least?

Reply:
[eb3e44d](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14175/commits/eb3e44db3ca401bd49a890e420613a3ab2f69023)